EmsgBox.go

```diff
@@ -0,0 +1,126 @@
+package mp4box
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+)
+
+//FullBox - Base box holding the bytes
```

Comment:
What is the structure of the box...
Please capture the payload structure

Reply:
https://aomediacodec.github.io/id3-emsg/

Reply:
updated the payload structure to the code.